modules/calypsoify/style-gutenberg.css

```diff
@@ -1,360 +1,4 @@
-html {
-  padding-top: 0px !important;
-}
-
-body.revision-php {
-	background: #f3f6f8;
-}
-
-#wpadminbar,
-#adminmenumain,
-#wp-admin-bar-menu-toggle {
+/* Hides the menu items for switching the view mode */
+.edit-post-more-menu__content .components-menu-group:first-child {
```

Comment:
This PR is more oriented at making `Calypsoified Gutenberg` and `Gutenberg in Calypso` as identical as possible, but in this particular case, I'd say let's keep the group visible even if it's not there in Calypso, and only hide the "Fullscreen Mode" item.

Also, notice that _now_ it's the last child, but that could change.
Off the top of my head there is no simple selector there, but I might be very wrong. 🙂

Reply:
Thanks for that! I've just committed a change handling this 🙂

I thought this view mode selector was like a radio button (you can only select one of them), but I realize that you can activate all of them at the same time, so it makes sense to only hide the "Fullscreen Mode" option.

Regarding the CSS selector using `:last-child`. I don't like it either, but the HTML rendered by Gutenberg doesn't provide any other class we can use for identifying this element:

<img width="689" alt="screen shot 2018-10-22 at 19 10 43" src="https://user-images.githubusercontent.com/1233880/47307375-2d807380-d62f-11e8-839e-0cb063e8a1b8.png">